Nadim.CinemaReservationSystem.Web/Startup.cs

```diff
@@ -58,18 +61,45 @@ public void ConfigureServices(IServiceCollection services)
         }
 
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
-        public void Configure(IApplicationBuilder app, IHostingEnvironment env)
+        public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerFactory loggerFactory)
         {
+            var filterLoggerSettings = new FilterLoggerSettings
+            {
+                {"Microsoft", LogLevel.Error },
+                {"System", LogLevel.Error },
+            };
+
+            loggerFactory = loggerFactory.WithFilter(filterLoggerSettings);
+            loggerFactory.AddFile(Configuration["Logging:LogFile"]);
+
             if (env.IsDevelopment())
             {
                 app.UseDeveloperExceptionPage();
             }
             else
             {
-                app.UseExceptionHandler("/Home/Error");
                 app.UseHsts();
             }
 
+            app.UseExceptionHandler(
```

Comment:
You should disable this in dev environment because it's helpful to get full exception info while developing